bug: I can't run "2024.6.1" using "npm run dev"

### Is this an issue specifically with Appium Inspector?

- [X] I have verified that my issue does _not_ occur with Appium, and should be investigated as an Appium Inspector issue

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I followed the steps below:

npm ci
npm run dev
However, Electron didn't start due to the following error:

After researching the error, I found that it was caused by the CJS loader being unable to find the "node
" module. The module path should be "internal/.../url.js".

Please help.

<error log>

internal/process/warning.js:43 (node:14464) UnhandledPromiseRejectionWarning: Error: Cannot find module 'node:url'
Require stack:
- C:\appium\appium-inspector\node_modules\glob\dist\commonjs\glob.js
- C:\appium\appium-inspector\node_modules\glob\dist\commonjs\index.js
- C:\appium\appium-inspector\node_modules\rimraf\dist\commonjs\index.js
- C:\appium\appium-inspector\node_modules\electron-extension-installer\dist\index.js
- C:\appium\appium-inspector\dist\main.js
- C:\appium\appium-inspector\node_modules\electron\dist\resources\default_app.asar\main.js
- 
    at Module._resolveFilename (internal/modules/cjs/loader.js:887:15)
    at Function.n._resolveFilename (electron/js2c/browser_init.js:257:1128)
    at Module._load (internal/modules/cjs/loader.js:732:27)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12913)
    at Module.require (internal/modules/cjs/loader.js:959:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (C:\appium\appium-inspector\node_modules\glob\dist\commonjs\glob.js:5:20)
    at Module._compile (internal/modules/cjs/loader.js:1078:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1108:10)
    at Module.load (internal/modules/cjs/loader.js:935:32)
(Use `electron --trace-warnings ...` to show where the warning was created)

### Expected Behavior

i expect "npm run dev". :-)....

### Operating System

Mac

### Appium Inspector Version

2024.6.1

### Appium Version

_No response_

### Further Information

Node.js: 20.12.2 or 18.20.3
OS: Windows 11 or macOS Sonoma
Git #: 984087dc73b12e79160fedffc5c42a57394d2736

--- a/app/electron/main/main.js
+++ b/app/electron/main/main.js
@@ -19,7 +19,7 @@ app.on('window-all-closed', () => {
 app.on('ready', () => {
   if (isDev) {
     require('electron-debug')();
-    // TODO: uncomment this after upgrading to newer Electron
+    // TODO: uncomment this after upgrading to Electron 15+
     // await installExtensions();
   }
 

--- a/app/electron/main/main.js
+++ b/app/electron/main/main.js
@@ -1,6 +1,6 @@
 import {app} from 'electron';
 
-import {installExtensions} from './debug';
+// import {installExtensions} from './debug';
 import {getAppiumSessionFilePath} from './helpers';
 import {setupMainWindow} from './windows';
 
@@ -16,10 +16,11 @@ app.on('window-all-closed', () => {
   app.quit();
 });
 
-app.on('ready', async () => {
+app.on('ready', () => {
   if (isDev) {
     require('electron-debug')();
-    await installExtensions();
+    // TODO: uncomment this after upgrading to newer Electron
+    // await installExtensions();
   }
 
   setupMainWindow({

